app/src/componentLibrary/tooltip/withTooltip/withTooltip.jsx

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2022 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useState } from 'react';
+import { Manager, Reference } from 'react-popper';
+import classNames from 'classnames/bind';
+import { Tooltip } from '../tooltip';
+import styles from './withTooltip.scss';
+
+const cx = classNames.bind(styles);
+const TOOLTIP_DELAY_MS = 2000;
```

Comment:
I'm not sure the user will wait 2s just to check if there is any tooltip on the element.
Can we shorten or even remove this delay?
Please discuss with UX.

Reply:
I discussed with UX, they said that this is Google's experience and it's OK

Reply:
Ok, let's wait for the feedback from our users on this delay.